lib/feature/one_hot_encoder.cc

```diff
@@ -91,16 +72,19 @@ OneHotEncoderModel::load(const std::string &path, tfrt::HostContext *host) {
   }
 
   std::ifstream model_data_input(path + "/data/" + model_data_filename);
-  std::string model_data_str((std::istreambuf_iterator<char>(model_data_input)),
-                             std::istreambuf_iterator<char>());
-  llvm::Error err = model->setModelData(std::move(model_data_str));
-  model_data_input.close();
-
-  if (err) {
+  if (!model->model_data_.ParseFromIstream(&model_data_input)) {
     return tfrt::MakeStringError(
-        "Failed to load OneHotEncoderModel: invalid model data file " + path +
+        "Failed to load OneHotEncoderModel: Invalid model data file " + path +
```

Comment:
String concatenation using the `+` operator could be inefficient due to the reasons explained in [1]. Given that we are already using `MakeStringError`, it seems better to use the following style. This style would also be useful in the future when we use non-string value in the error message.

Same for other usages of `MakeStringError`.

```
return tfrt::MakeStringError(
    "Failed to load OneHotEncoderModel: Invalid model data file ", path, "/data/" + model_data_filename);
```

[1] https://stackoverflow.com/questions/611263/efficient-string-concatenation-in-c

Reply:
OK. I'll make the change to all usages of `MakeStringError`.

Apart from `MakeStringError`, there are also many other usages of string concatenation. I also changed all these to `tfrt::StrCat`, which is how `MakeStringError` concatenate the arguments while preserving efficiency.